src/root/components/PdfTextOutput/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Text, View } from '@react-pdf/renderer';
+import React from 'react';
+import pdfStyles from 'src/styles/pdf/pdfStyles';
+
+export function PdfTextOutput(props: {
+  label: string;
+  value?: string;
+  columns?: '1/3' | '2/3' | '3/3' | '1/2';
+}) {
+  const {
+    label,
+    value,
+    columns = '1/3',
+  } = props;
+
+  const styleMap = {
+    '1/3': pdfStyles.oneByThree,
+    '2/3': pdfStyles.twoByThree,
+    '3/3': pdfStyles.threeByThree,
```

Comment:
Isn't this just `1`?

Reply:
not always as we have to make columns of different sizes